docs/changelog-fragments/621.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed reading large files over SCP -- by :user:`Jakuje`
```

Comment:
Please, add a trailing period to the sentence. Also, it'd be nice to elaborate on this in a way that it would prompt the users of what changed in the behavior compared to the previous release. High-level, not need to be very detailed.